Makefile

```diff
@@ -154,41 +196,90 @@ $(PROTO_MODEL): $(OUTPUT_DIR)/unicharset data/radical-stroke.txt
 	combine_lang_model \
 	  --input_unicharset $(OUTPUT_DIR)/unicharset \
 	  --script_dir data \
+	  --numbers $(NUMBERS_FILE) \
+	  --puncs $(PUNC_FILE) \
+	  --words $(WORDLIST_FILE) \
```

Comment:
Can they be empty? Personally, I do not want word lists in my OCR models.

Reply:
If the file is not there, it should ignore. However, I will verify with a test and get back.

Reply:
It gives a message and continues.

```
combine_lang_model \
  --input_unicharset data/eng/unicharset \
  --script_dir data \
  --numbers data/eng/eng.numbers \
  --puncs data/eng/eng.punc \
  --words data/eng/eng.wordlist \
  --output_dir data \
   \
  --lang eng
Failed to read data from: data/eng/eng.wordlist
Failed to read data from: data/eng/eng.punc
Failed to read data from: data/eng/eng.numbers
Loaded unicharset of size 113 from file data/eng/unicharset
Setting unichar properties
Other case É of é is not in unicharset
Other case FI of fi is not in unicharset
Setting script properties
Warning: properties incomplete for index 47 = ~
Config file is optional, continuing...
Failed to read data from: data/eng/eng.config
Null char=2
lstmtraining \
  --debug_interval 0 \
  --traineddata data/eng/eng.traineddata \
  --old_traineddata /home/ubuntu/tessdata_best/eng.traineddata \
  --continue_from data/eng/eng.lstm \
  --model_output data/eng/checkpoints/engPlus \
  --train_listfile data/eng/list.train \
  --eval_listfile data/eng/list.eval \
  --max_iterations 4000
Loaded file data/eng/eng.lstm, unpacking...
Warning: LSTMTrainer deserialized an LSTMRecognizer!
Code range changed from 111 to 111!
```